image/README.md

```diff
@@ -1,14 +1,10 @@
-# Imaging
```

Comment:
Algorithms files should be in its own folder and not in image folder. Move Readme file. Dependencies list missing

Reply:
Thanks, Shakes. I have had that problem fixed!